frequency-chain/frequency-xyz
# 37 update theme with the one from the style guide

# Description

Update theme to use the styleguide.

- Closes #37 

## Type of change


# How to Test?

Pull branch and see that styles look the same and are not broken in any screen size.

1. Pull and run locally
2. See that css is coming through as well as the theme. Best examples are the underline on hover in the nav (showing css) and the colors properly coming through (tailwind config)

# Checklist:
